cmd/credential-helper/main.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+func main() {
+	password := os.Getenv("GIT_PASSWORD")
+	if password == "" {
+		fmt.Fprintln(os.Stderr, "GIT_PASSWORD must be set")
+		os.Exit(1)
+	}
+	fmt.Println(password)
```

Comment:
I assume it does not matter `fmt.Println` ends with `\n`?

Reply:
Does not seem to. This works.